driver-core/src/main/java/com/datastax/driver/core/SettableData.java

```diff
@@ -16,7 +16,7 @@
 package com.datastax.driver.core;
 
 /**
- * Collection of (typed) CQL values that can set either by index (starting a 0) or by name.
+ * Collection of (typed) CQL values that can set either by index (starting at zero) or by name.
```

Comment:
"that can **be** set" again – I suppose we need to amend SettableByNameData as well.